crud/users.py

```diff
@@ -0,0 +1,57 @@
+from sqlalchemy import select
+
+from db import Session
+from models import User
+
+
+def create(user: User):
+    with Session.begin() as session:
+        try:
+            session.add(user)
+        except:
```

Comment:
```suggestion
        except Exception:
```
Do not use bare except. Ideally also log exception somewhere, but that's for the future